Integrations/CentralBankIntegrationLib/CentralBankCurrenciesPublisher.cs

```diff
@@ -0,0 +1,66 @@
+﻿using Interfaces;
+using System.Collections.Generic;
+using System.Net;
+using System.IO;
+using System.Xml.Serialization;
+using System.Text;
+using CentralBankIntegrationLib.XmlSerializationObjects;
+using System.Linq;
+using System;
+using DTO;
+using System.Threading.Tasks;
+
+namespace CentralBankIntegration
+{
+    /// <summary>
+    /// Get reference information from Central Bank.
+    /// </summary>
+    public class CentralBankCurrenciesPublisher : INewsPublisher
+    {
+        const string uri = "http://www.cbr.ru/scripts/XML_daily.asp";
+        DataFromCentralBank data;
+
+        public CentralBankCurrenciesPublisher(Uri uriBank)
+        {
+            data = GetData(uriBank);
+        }
+
+        public CentralBankCurrenciesPublisher() : this(new Uri(uri)) { }
+
+        public List<ExchangeRate> GetCurrencies()
+        {
+            List<ExchangeRate> rates = new List<ExchangeRate>();
+            //Parallel.For(0, data.Currencies.Count, (int i) =>
```

Comment:
Why do we want to  have comments in develop branch?

Reply:
Deleted

Reply:
Deleted